x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/users_empty_prompt.tsx

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import {
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPanel,
+  EuiEmptyPrompt,
+  EuiLink,
+  EuiSpacer,
+} from '@elastic/eui';
+
+import { USERS_DOCS_URL } from '../../enterprise_search/routes';
+
+import { NO_USERS_TITLE, NO_USERS_DESCRIPTION, ENABLE_USERS_LINK } from './constants';
+
+export const UsersEmptyPrompt: React.FC = () => (
+  <EuiFlexGroup alignItems="center" justifyContent="center">
+    <EuiFlexItem>
+      <EuiSpacer />
+      <EuiPanel style={{ maxWidth: 700, margin: '0 auto' }}>
```

Comment:
[optional] any chance we can make this a CSS class instead? 🤷

Reply:
Also, I don't _think_ you need the wrapping `<EuiFlexGroup>`

Reply:
Choosing my battles with 3 days before FF. Added to Google doc for tech debt after 7.14 FF